docs/book/src/cronjob-tutorial/testdata/project/config/default/kustomization.yaml

```diff
@@ -25,17 +25,11 @@ resources:
 - ../certmanager
 # [PROMETHEUS] To enable prometheus monitor, uncomment all sections with 'PROMETHEUS'.
 - ../prometheus
-# [METRICS] To enable the controller manager metrics service, uncomment the following line.
-#- metrics_service.yaml
+# [METRICS] Expose the controller manager metrics service.
+- metrics_service.yaml
```

Comment:
Hi @joelanford 

IHMO it should not be enable by default at all.
WHY?

- Not everybody uses those metrics and care about.
- Users should be aware of the risks and take the explicit decision of the should or not expose the metrics
- Note that, we probably will see a lot of people impacted by https://github.com/kubernetes-sigs/kubebuilder/discussions/3907 only because it was enabled by default, but they did not were aware of or using it at all.

Reply:
Yeah, I waffled on this. In 3.14.x and earlier, we had the following:
- The Kube RBAC proxy patch was enabled by default, which meant that the default scaffold would run a secure KRP proxy to a plaintext and localhost metrics server running in the controller process.
- The Role and Role binding that gave KRP permissions were enabled by default.
- The metrics service was enabled by default.

Because of that, I landed on the side of the fence of "turn it all on by default" to try to keep the behavior as close to the same as possible.

But I see the argument (especially in kubebuilder/v4) of turning it all off by default.